manuscript/setup/dev.md

```diff
@@ -127,12 +136,12 @@ export INGRESS_HOST=$(kubectl --namespace traefik \
     get service traefik \
     --output jsonpath="{.status.loadBalancer.ingress[0].ip}")
 
-# If EKS
+# If YES EKS
```

Comment:
The original is cryptic, but this seem to be even more cryptic. Something like, "Execute the command that follows ONLY if you are using EKS."

Reply:
Love it. Fixed it. Go team!